study-builder/studies/testboston/symptoms-survey.conf

```diff
@@ -0,0 +1,50 @@
+{
+  include required("../snippets/activity-general-form.conf"),
+  "studyGuid": ${id.study},
+  "activityCode": ${id.act.symptoms_survey},
+  "versionTag": "v1",
+  "displayOrder": 4,
+  "writeOnce": true,
+  "maxInstancesPerUser": 1,
+  "translatedNames": [
+    { "language": "en", "text": ${i18n.en.symptoms_survey.name} },
+    { "language": "es", "text": ${i18n.es.symptoms_survey.name} },
+    { "language": "ht", "text": ${i18n.ht.symptoms_survey.name} }
+  ],
+  "translatedTitles": [
+    { "language": "en", "text": ${i18n.en.symptoms_survey.title} },
+    { "language": "es", "text": ${i18n.es.symptoms_survey.title} },
+    { "language": "ht", "text": ${i18n.ht.symptoms_survey.title} }
+  ],
+  "sections": [
+    {
+      "nameTemplate": null,
+      "icons": [],
+      "blocks": [
+        {
+          "control": ${_includes.symptoms_survey.baseline_symptoms},
+          "nested": [
+            {
+              "question": ${_includes.symptoms_survey.symptoms_start_date},
+              "blockType": "QUESTION",
+              "shownExpr": """user.studies["testboston"].forms["SYMPTOMS_SURVEY"].questions["BASELINE_SYMPTOMS"].answers.hasAnyOption("COUGH", "FEVER", "SORE_THROAT", "SHORT_BREATH", "TIGHT_CHEST", "FATIGUE", "MUSCLE_ACHE", "SENSORY_LOSS", "NAUSEA", "CONGESTION", "HEADACHE")
```

Comment:
Might be easier to use negation instead:
```
!user...hasOption("NONE")
```

Reply:
With that question is displayed when user doesn't choose any option.

Reply:
How about:
```
user...question..isAnswered() && !user...hasOption("NONE")
```